rasa/core/tracker_store.py

```diff
@@ -643,19 +643,19 @@ def get_db_url(
         from sqlalchemy.engine.url import URL
 
         # Users might specify a url in the host
-        parsed = urlsplit(host or "")
-        if parsed.scheme:
+        if host and "://" in host:
```

Comment:
if fixed it a bit differently (currently on the .x branch), please check which one you prefer

Reply:
https://github.com/RasaHQ/rasa/pull/5222/files